docs/guides/validation/validate_data_overview.md

```diff
@@ -9,4 +9,48 @@ import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
 
 <UniversalMap setup='inactive' connect='inactive' create='inactive' validate='active'/>
 
-** This is a placeholder until content is created, so that corresponding links don't cause docusaurus to break **
+When you have completed this step for the first time, you will have created and run a Checkpoint.  This Checkpoint can then be reused to Validate data in the future, and you can also create and configure additional Checkpoints to cover different use cases, should you have them.
+
+## The Validate Data process
+
+The recommended workflow for validating data is through **the use of Checkpoints.**  Checkpoints handle the rest of the validation process for you: They will validate data, save Validation Results, run any Actions you have specified, and finally create Data Docs with their results.  Once you have created your Checkpoint, configured it to your specifications, and specified any Actions you want it to take all you will need to do is tell the Checkpoint to run.
+
+### Creating a Checkpoint
+
+Checkpoints are simple to create.  While advanced users could write their configuration from scratch, we recommend using the CLI.  It will launch a Jupyter Notebook set up with boilerplate code to create your checkpoint.  All you will need to do is configure it!  For detailed instructions, please see our guide on [how to create a new Checkpoint](./checkpoints/how_to_create_a_new_checkpoint.md).
+
+### Configuring your Checkpoint
+
+There are three very important things you can do when configuring your Checkpoint.  You can add additional validation data, or set the checkpoint so that validation must be specified at run time.  You can add additional Expectation Suites, and you can add Actions which the Checkpoint will execute when it finishes validating data.  For a more detailed overview of Checkpoint configuration, please see our documentation on [Checkpoints and Actions](../../reference/checkpoints_and_actions.md).
```

Comment:
Tiny nit: since you specify that there are three things, could you number the things?
```suggestion
There are three very important things you can do when configuring your Checkpoint.  1) You can add additional validation data, or set the checkpoint so that validation must be specified at run time.  2) You can add additional Expectation Suites, and 3) you can add Actions which the Checkpoint will execute when it finishes validating data.  For a more detailed overview of Checkpoint configuration, please see our documentation on [Checkpoints and Actions](../../reference/checkpoints_and_actions.md).
```

Reply:
I had considered this, but using direct numbers makes it feel like a collapsed ordered list rather than a paragraph (and an ordered list didn't feel right given the length of some of the items.  It may be better to use First, Second, and finally, instead of 1) 2) 3); I'll make those as the change instead.